src/metrics_server/connection_metrics.ts

```diff
@@ -45,18 +50,28 @@ function getConnectionRowsFromReport(report: HourlyConnectionMetricsReport): Con
   const endTimestampStr = new Date(report.endUtcMs).toISOString();
   const rows = [];
   for (const userReport of report.userReports) {
+    if (!isCurrentUserReport(userReport)) {
```

Comment:
I'd rather revert to the previous behavior, it's clearer, because "current" is not well defined. The semantic of the old report should still be valid.
I believe we checked if it had the countries field, and reported it as a country report, but look at the old code.

Reply:
Are you suggesting we should continue inserting the access key reports? I thought the point was to stop collecting those. We need to drop them somewhere.

Reply:
I think this would be clearer and backward compatible:
```suggestion
    if (userReport.countries) {
```

It's effectively checking "is this a usage per location report?" (vs a "usage per key report").

It's based on what it is or has. The concept of "current" is not clear. It's also potentially wrong. In the past we had reports with both user id and countries in the same report. We shouldn't drop those, we just need to ignore the user id